reference/configuration/framework.rst

```diff
@@ -1478,7 +1483,45 @@ enabled
 
 Whether to enable the ``serializer`` service or not in the service container.
 
-For more details, see :doc:`/cookbook/serializer`.
+.. _reference-serializer-cache:
+
+cache
+.....
+
+**type**: ``string``
+
+The service that is used to persist class metadata in a cache. The service
+has to implement the :class:`Doctrine\\Common\\Cache\\Cache` interface.
+
+.. seealso::
+
+    For more information, see :ref:`cookbook-serializer-enabling-metadata-cache`.
+
+.. _reference-serializer-enable_annotations:
+
+enable_annotations
+..................
+
+**type**: ``boolean`` **default**: ``false``
+
+If this option is enabled, serialization groups can be defined using annotations.
+
+.. seealso::
+
+    For more information, see :ref:`cookbook-serializer-using-serialization-groups-annotations`.
+
+name_converter
+..............
+
+**type**: ``string``
+
+The name converter to use.
+The :class:`Symfony\\Component\\Serializer\\NameConverter\\CamelCaseToSnakeCaseNameConverter` name converter can enabled by using the ``serializer.name_converter.camel_case_to_snake_case``
+value.
+
+.. seealso::
+
+    For more information, see :ref:`component-serializer-converting-property-names-when-serializing-and-deserializing`.
```

Comment:
same here (the ref should be placed on its own line). I would also prefer if the ref pointer can be a little bit smaller.